packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -46,16 +46,17 @@ describe("Integration tests with bov+tendermint", () => {
   // TODO: had issues with websockets? check again later, maybe they need to close at end?
   // max open connections??? (but 900 by default)
   const tendermintUrl = "ws://localhost:22345";
+  const entry = Ed25519HdWallet.fromMnemonic(mnemonic);
```

Comment:
This shares one mutable `Ed25519HdWallet` across all the specs. Thus we should keep it locally